### PR TITLE
Fix the association of code coverage results with source code 

### PR DESCRIPTION
**Pull request by peternied**
### Description
Fix the association of code coverage results with source code.  You'll have seen how this was broken because we stopped getting the comment on PRs with coverage information.  My apologies for breaking this in the most recent change to centralized the CC runs.

### Issues Resolved
- Resolves https://github.com/opensearch-project/security/issues/2649

### Testing
✅ Inspection CC report on this PR, see [[link]](https://github.com/opensearch-project/security/pull/3668#issuecomment-1802489151).

### Check List
- [ ] ~New functionality includes testing~
- [ ] ~New functionality has been documented~
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by willyborankin — APPROVED** [review]

**Comment by peternied** [comment]
@RyanL1997 @DarshitChanpura Could I get a second review?

**Review by scrawfor99 — APPROVED** [review]

**Pull request merged**
